modules/nf-core/krakenuniq/preloadedkrakenuniq/main.nf

```diff
@@ -30,13 +29,13 @@ process KRAKENUNIQ_PRELOADEDKRAKENUNIQ {
     def args = task.ext.args ?: ''
     def args2 = task.ext.args ?: ''
 
-    def classified   = meta.single_end ? '"\${PREFIX}.classified.fastq"'   : '"\${PREFIX}.classified#.fastq"'
-    def unclassified = meta.single_end ? '"\${PREFIX}.unclassified.fastq"' : '"\${PREFIX}.unclassified#.fastq"'
-    def classified_option = save_output_fastqs ? "--classified-out ${classified}" : ''
-    def unclassified_option = save_output_fastqs ? "--unclassified-out ${unclassified}" : ''
+    def classified   = meta.single_end ? '"\${PREFIX}.classified.fasta"'   : '"\${PREFIX}.merged.classified.fasta"'
+    def unclassified = meta.single_end ? '"\${PREFIX}.unclassified.fasta"' : '"\${PREFIX}.merged.unclassified.fasta"'
```

Comment:
Are they actually merged? I wonder if we need to include that in the file name... it might confuse people that the module is doin some internal module (as in, outside centrifuge itself), I think I would rather have just `un/classified.fasta` as sufficent. As far as I can see from your comments there still is not any difference between single or paired end output?

Reply:
Not sure what centrifuge has to do with this.

When the input is paired-end, the reads are merged and output into a single FASTA file. So it makes sense to me to name it that way.

Reply:
Also, see my comment here https://github.com/nf-core/taxprofiler/issues/420#issuecomment-1817625086